.golangci.yml

```diff
@@ -23,6 +23,7 @@ linters:
     - nolintlint
     - revive
     - staticcheck
+    # - testifylint
```

Comment:
Why is this commented out here? Is it just because it doesn't fix everything?

Reply:
Yes, this is only the first batch